### ISSUE TITLE
OrderRefund.amount data type

### ISSUE THREAD

**Issue by Outzzz** [XiaoFaye/WooCommerce.NET#412]
### Make sure you have included the below details when open an issue. Thank you.

- Wordpress  5.2.3, WooCommerce 3.7.0 and WooCommerce.NET 0.7.9

- Steps to replicate the issue
```
OrderRefund orderRefund = new OrderRefund {
    reason = "refund",
    amount = 1,
};

await WCObject.Order.Refunds.Add(orderRefund, orderID).ConfigureAwait(false);
```
- Details of the error message if there is any

```
{
    "code": "rest_invalid_param",
    "message": "无效参数：amount",
    "data": {
        "status": 400,
        "params": {
            "amount": "amount的类型不是string。"
        }
    }
}
```
在[官方文档](https://woocommerce.github.io/woocommerce-rest-api-docs/#refunds)里, amount是string

**Comment by XiaoFaye** [comment]
Hi, thanks for raising this issue, there is a bug in the OrderRefund class, it will be fixed in the next version.

**Comment by Outzzz** [comment]
this is alternative way to refund
```
OrderRefund orderRefund = new OrderRefund {
    reason = "refund",
};

await WCObject.Order.Refunds.Add(orderRefund, orderID, new Dictionary<string, string> { { "amount", "1" } }).ConfigureAwait(false);
```
however, i try my best but still cant add a line item refund via rest API.
does OrderRefund.line_item works? can you give me some addvice or sample.
i can provide testing envirment if you need

**Comment by XiaoFaye** [comment]
Can you add the refund in this way?
```cs
OrderRefund orderRefund = new OrderRefund {
    reason = "refund",
};

await WCObject.Order.Refunds.Add(orderRefund, orderID).ConfigureAwait(false);
```
If you can, then it's possible to update the amount as below:

```cs
await WCObject.Order.Refunds.UpdateWithNull(refundID, orderID, new { amount = "12.50" }).ConfigureAwait(false);
```

Of course this just a temporary solution before the new version is released.

**Comment by Outzzz** [comment]
yes.that is what i am using now.you can do that in one request."amount" parameter is valid while you create a refund

**Comment by XiaoFaye** [comment]
Hi @Outzzz , this issue is fixed in the version 0.8.0
